### PR TITLE
feat(SkipLink): New component

### LINKED ISSUE #1055: Skip link: Develop react component

### PR DESCRIPTION
**Pull request by stianmorsund**
Resolves #1055

### CODE REVIEW

File: packages/react/src/components/SkipLink/SkipLink.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import utilityClasses from './../../utilities/utility.module.css';
+import classes from './SkipLink.module.css';
+
+export type SkipLinkProps = {
+  /** The content to display inside the skiplink. */
+  children: ReactNode;
+
+  /** Custom class name for the skiplink. This will be appended to the design system class names. */
+  className?: string;
```
Comment (Barsnes) [inline packages/react/src/components/SkipLink/SkipLink.tsx:13]: You don't need to put classname here, you get it from `...rest`.

File: packages/react/src/components/SkipLink/SkipLink.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  /** Href of an element in the DOM the skiplink should skip to. E.g #main-content */
+  href?: string;
```
Comment (Barsnes) [inline packages/react/src/components/SkipLink/SkipLink.tsx:16]: This should be required

File: packages/react/src/components/SkipLink/SkipLink.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+import { SkipLink } from '.';
+type Story = StoryObj<typeof SkipLink>;
+
+export default {
+  title: 'Felles/SkipLink',
+  component: SkipLink,
+} as Meta;
+
+export const Preview: Story = () => (
```
Comment (Barsnes) [inline packages/react/src/components/SkipLink/SkipLink.stories.tsx:12]: You have a type error here:
```
src/components/SkipLink/SkipLink.stories.tsx(12,14): error TS2559: Type '() => Element' has no properties in common with type 'StoryAnnotations<ReactRenderer, { children: ReactNode; className?: string | undefined; href?: string | undefined; } & AnchorHTMLAttributes<HTMLAnchorElement>>'.
```

File: packages/react/src/components/SkipLink/SkipLink.module.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  height: auto;
+  margin: inherit;
+  overflow: visible;
+  clip: auto;
+  clip-path: none;
+  white-space: inherit;
+  text-decoration: underline;
+  text-decoration-thickness: 3px;
+  text-underline-offset: 0.3em;
+  padding: 15px;
```
Comment (Barsnes) [inline packages/react/src/components/SkipLink/SkipLink.module.css:15]: Padding is different in figma, you should be using spacing tokens

**Review by Barsnes — CHANGES_REQUESTED** [review]
Looks good ✨ 
Commented on some minor things that needs tweaking

**Review by mimarz — APPROVED** [review]
Awesomesauce! This looks good to me 😊 Thank you for contributing to development of our components, much appreciated! 👏 

awesome-sauce-david-tennant

**Pull request merged**